packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -151,6 +151,18 @@ export class Loan {
     return await this.client.call('setloantoken', [{ ...defaultData, ...loanToken }, utxos], 'number')
   }
 
+  /**
+   * Quick access to multiple API with consolidated total collateral and loan value.
+   * 1. @see {@link listCollateralTokens}
+   * 2. @see {@link listLoanTokens}
+   * 3. @see {@link listLoanSchemes}
```

Comment:
```suggestion
   * @see {@link listCollateralTokens}
   * @see {@link listLoanTokens}
   * @see {@link listLoanSchemes}
```

the `1.` doesn't work well with `@see` symbol

Reply:
Render Docs (using IntelliJ you can see it will look on JSDoc processors)

![image](https://user-images.githubusercontent.com/4266087/135578696-fb244e90-cb66-46ca-8aa9-e816d32c9034.png)

after

![image](https://user-images.githubusercontent.com/4266087/135578729-a7950a3b-1049-4ad3-a34f-c9d84d66e0f3.png)